app/DateTimePicker/time-picker.scss

```diff
@@ -4,38 +4,38 @@
     &.padding {
         padding: 16px;
     }
-    .picker-time-icon {
-        position: absolute;
-        top: 4px;
-        right: 2px;
-        cursor: pointer;
-        padding-right: 0.8rem;
+    span {
```

Comment:
This will compile to css and apply this styles to all `spans` in the app. Could you try to use more specific selector?